### ISSUE TITLE
Generate Citation Guideline section in NHA report template

### ISSUE THREAD

**Issue by Anna-Johnson** [PNHP/NHA_Tools#31]
To direct report users in how we would like them to cite the report, automatically generate the correct citation format for them to use at the bottom of each report. 

I would suggest something like this--kind of an "article within an encyclopedia volume" approach:

Pennsylvania Natural Heritage Program. ([Year]). [Site Name] NHA. In Pennsylvania Natural Heritage Inventories ([County Name], page [##]). Available at: http://www.naturalheritage.state.pa.us/CNHI.aspx

**Comment by ChristopherTracey** [comment]
Not bad, although instead of a county report, we may just want to reference the 
"Grand Database of Natural Heritage Areas" or whatever we plan on calling it.

**Comment by Anna-Johnson** [comment]
Yes, that could also work--treat each  NHA as a single entry into the big database, if we're gently trying to move away from county as the primary unit...
